.github/workflows/stale-pr.yml

```diff
@@ -0,0 +1,16 @@
+name: "Close stale pull requests"
+on:
+  schedule:
+  - cron: "0 0 * * *"
```

Comment:
this runs once a day? (could be more frequent while in debug mode)

also could add https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events as a comment.

Reply:
How about every 6 hours during debugging?

Reply:
We can easily run builds manually in GitHub actions afaik. I'd say once a day is good enough, also for debug, but doesn't make a difference.